flutter/pubspec.yaml

```
@@ -16,7 +16,7 @@ dependencies:
   flutter_web_plugins:
     sdk: flutter
   sentry: 7.18.0
-  package_info_plus: '>=1.0.0 <=5.0.1'
+  package_info_plus: '>=1.0.0 <=6.0.0'
```

Shouldn't it be the following since there's already v6?
```suggestion
  package_info_plus: '>=1.0.0 <=7.0.0'
```

---

7.0.0 does not exist and might contain breaking changes once released. We could probably go with `<7.0.0` if we assume 6.x version will not have breaking changes.

---

Oh yeah, you're right. `<7.0.0` is what I meant